os/android/hwcservice.h

```diff
@@ -97,6 +97,11 @@ class HwcService : public BnService {
 
     status_t DisableHDCPSessionForAllDisplays();
 
+#ifdef ENABLE_PANORAMA
+    status_t EnablePanorama(uint32_t hotplug_simulation);
```

Comment:
suggest change the name enable/disable to Trigger/Shutdown if you have to keep two APIs.

Reply:
The API naming has been revised accordingly in latest commit.